src/main.go

```diff
@@ -352,6 +467,101 @@ func (t *tester) addSuccess(testSuite *XUnitTestSuite, startTime *time.Time, cnt
 	})
 }
 
+func generateBRStatements(source, target string) (string, string) {
+	// Generate a random UUID
+	uuid := uuid.NewString()
+
+	// Create the TMP_DIR path
+	tmpDir := fmt.Sprintf("/tmp/%s_%s", source, uuid)
+
+	// Generate the SQL statements
+	backupSQL := fmt.Sprintf("BACKUP TABLE `%s` TO '%s'", source, tmpDir)
```

Comment:
It seems that already supports `backup` and `restore` SQL, maybe we can use SQL directly instead of use comments.

Reply:
Yeah, when I added 'backup_and_restore' command, I was thinking `br` binaries. However, it might be OK to just use SQL now and only try `br` binary when needed.

Reply:
mmm...it is not straightforward to use SQL directly:

```
BACKUP TABLE T TO 'LOCAL:///TMP/BR';
Destination	Size	BackupTS	Queue Time	Execution Time
local:///TMP/BR	3687	453115446250176572	2024-10-09 11:24:13	2024-10-09 11:24:13
DROP TABLE T;
RESTORE TABLE T FROM 'LOCAL:///TMP/BR';
Destination	Size	BackupTS	Cluster TS	Queue Time	Execution Time
local:///TMP/BR	3687	453115446250176572	453115447167680565	2024-10-09 11:24:17	2024-10-09 11:24:17
```
The return rows of `BACKUP` and `RESTORE` are time-specific.

Reply:
we could use `--disable_result_log` to disable print return rows. I think decoupling these functions will make it more convenient to use.